src/index.css

```
@@ -36,6 +49,13 @@
 
   body {
     @apply min-h-screen m-0;
+    /* Base background color */
+    background-color: rgb(var(--surface-base));
+    /* Subtle radial gradient overlay for depth */
+    background-image:
+      radial-gradient(ellipse 80% 50% at 50% -20%, rgb(var(--accent-500) / 0.08), transparent 50%),
```

Stylelint (17.2.0)

[error] Expected "0.08" to be "8%" (alpha-value-notation)

(alpha-value-notation)